pyEpiabm/pyEpiabm/update_place_sweep.py

```diff
@@ -0,0 +1,38 @@
+#
+# Sweep to update people present in a place
+#
+from .abstract_sweep import AbstractSweep
+import random
+import numpy as np
+
+
+class UpdatePlaceSweep(AbstractSweep):
+    """Class to update people in the "Place"
+    class.
+    """
+    def __call__(self, time: int):
+        """Given a population structure, updates the people
+        present in each place at a specific timepoint.
+
+        :param time: Current simulation time
+        :type time: int
+        """
+        # Double loop over the whole population, clearing places
+        # and refilling them.
+        for cell in self._population.cells:
+            for place in cell.places:
+                place.empty_place()
+                # Possibly want to later differentiate between
+                # the fixed population of a place ie staff, and
+                # the variable population ie customers.
+
+                # Ensure that the number of people put in the place
+                # is at most its capacity or the total number of
+                # people in the microcell.
+                max_capacity = np.minimum(place.max_capacity,
+                                          len(place.microcell.persons))
+
+                new_capacity = random.randint(1, max_capacity)
+                for _ in range(new_capacity):
+                    i = random.randint(1, len(place.microcell.persons))
+                    place.add_person(place.microcell.persons[i-1])
```

Comment:
Do we need a check here to ensure the person is not already in the place? Couldn't see anything in the place class to ensure that a person can't be added to the same place twice (or indeed can only be in one place at a time) but I might be wrong - lots of code here!

Reply:
Good idea to have a check they're not already there. Not sure being in the same place twice is a bad thing? Timestep is still a day so very feasible people might visit multiple places.

Reply:
Okay that sounds good! I still feel like a person being in two places simultaneously is non intuitive, so if we can document that we have done this that would be great!